docs/userguide/containers/dockervolumes.md

```diff
@@ -144,7 +144,7 @@ Mounting a host directory can be useful for testing. For example, you can mount
 source code inside a container. Then, change the source code and see its effect
 on the application in real time. The directory on the host must be specified as
 an absolute path and if the directory doesn't exist the Engine daemon automatically
-creates it for you.  This auto-creation of the host path has been [*deprecated*](#auto-creating-missing-host-paths-for-bind-mounts).
+creates it for you. This auto-creation of the host path has been [*deprecated*](../../../deprecated/#auto-creating-missing-host-paths-for-bind-mounts).
```

Comment:
can you change this to `../../deprecated.md#auto-creating-missing-host-paths-for-bind-mounts`? That should make it work both on GitHub *and* in the documentation; (one less `../` and `deprecated.md` in stead of `deprecated/`.

you can test the docs locally by running `make docs` - that will start a container and build the docs inside that container and start a web server to test it